webpack/localization-plugin/src/WebpackConfigurationUpdater.ts

```diff
@@ -113,13 +109,9 @@ export class WebpackConfigurationUpdater {
               Constants.RESX_REGEX
             ]
           },
-          use: [
-            require.resolve('json-loader'),
-            {
-              loader: loader,
-              options: loaderOptions
-            }
-          ]
+          loader: loader,
+          options: loaderOptions,
```

Comment:
Apparently these options were deprecated in favor of use: `[ { loader: loader, options: loaderOptions } ]` so may be best to stick with the old format on that